src/applications/personalization/profile/util/direct-deposit.js

```diff
@@ -0,0 +1,55 @@
+import cloneDeep from 'lodash/cloneDeep';
+import set from 'lodash/set';
+
+export class DirectDepositClient {
+  #PPIU_ENDPOINT = '/ppiu/payment_information';
+
+  #LH_ENDPOINT = '/profile/direct_deposits/disability_compensations';
+
+  constructor({ useLighthouseDirectDepositEndpoint = false }) {
+    this.useLighthouseEndpoint = useLighthouseDirectDepositEndpoint;
+  }
+
+  get endpoint() {
+    return this.useLighthouseEndpoint ? this.#LH_ENDPOINT : this.#PPIU_ENDPOINT;
+  }
+
+  generateApiRequestOptions(payload) {
```

Comment:
would it make sense to change `payload` to `fields` like you did in BankInfo.jsx? (just to stay consistent)